pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y)
```

Comment:
There's a colon missing at the end of the function definition

Reply:
Got it!